### PR TITLE
Update documentation & example for flexvolume.

### PR DESCRIPTION
**Pull request by chakri-nelluri**
**What this PR does / why we need it**:
Updates documentation for latest Flex Volume changes in #26926

**Which issue this PR fixes** _(optional, in `fixes #<issue number>(, #<issue_number>, ...)` format, will close that issue when PR gets merged)_: fixes #

**Special notes for your reviewer**:

**Release note**:

``` release-note
```

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/30853)

### CODE REVIEW

**Comment by chakri-nelluri** [comment]
@MikaelCluseau @saad-ali @rootfs PTAL Thanks..

File: examples/volumes/flexvolume/README.md [hunk 1]
```diff
@@ -29,55 +29,103 @@ Documentation for other releases can be found at
...
 
-It allows for vendors to develop their own drivers to mount volumes on nodes.
+It allows vendors to develop their own drivers and expose their volumes to Kubernetes pods.
 
 *Note: Flexvolume is an alpha feature and is most likely to change in future*
 
 ## Prerequisites
 
-Install the vendor driver on all nodes in the kubelet plugin path. Path for installing the plugin: /usr/libexec/kubernetes/kubelet-plugins/volume/exec/\<vendor~driver\>/\<driver\>
+Install the vendor driver on all nodes(including master nodes) in the Kubelet plugin path. Path for installing the plugin: /usr/libexec/kubernetes/kubelet-plugins/volume/exec/\<vendor~driver\>/\<driver\>
```
Comment (MikaelCluseau) [inline examples/volumes/flexvolume/README.md]: there should be a space before "("

**Comment by MikaelCluseau** [comment]
LGTM

File: examples/volumes/flexvolume/README.md (hunk 1 above)
Comment (chakri-nelluri) [inline examples/volumes/flexvolume/README.md]: ACK

File: examples/volumes/flexvolume/README.md [hunk 2]
```diff
@@ -88,7 +136,8 @@ following format.
 {
 	"status": "<Success/Failure>",
```
Comment (MikaelCluseau) [inline examples/volumes/flexvolume/README.md]: we also have "Not supported"

**Comment by thockin** [comment]
ok to test

**Comment by chakri-nelluri** [comment]
Closing this request as the flexvolume rework didn't make it to 1.4. New API proposal will be in 1.5+

**Pull request closed without merging**